java/java.navigation/src/org/netbeans/modules/java/navigation/ElementNode.java

```diff
@@ -309,7 +309,7 @@ public void updateRecursively( Description newDescription ) {
         }
     }
     
-    public Description getDescritption() {
+    public Description getDescription() {
```

Comment:
note to self: checked and `java.navigation` does not export this package (or any other package). `Description` itself is package private interestingly.

@matthiasblaesing can we make this method package private?